web/crux-ui/src/elements/dyo-chips.tsx

```diff
@@ -1,27 +1,31 @@
 import assert from 'assert'
 import clsx from 'clsx'
+import useTranslation from 'next-translate/useTranslation'
 import { useState } from 'react'
 
 interface DyoChipsProps<T> {
   className?: string
   key?: React.Key
   choices: readonly T[]
+  isFilter?: boolean
```

Comment:
Let's be explicit about the implication of this prop. Rename it to addAllOption or something